File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L5/04/Kasus2.ts
import * as readline from 'readline';

function hitungArea(r: number): number {
    return r * r * Math.PI;
}

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Enter the radius and length of a cylinder: ', (input) => {
    const [r, l] = input.split(' ').map(Number);
    console.log('The area is', hitungArea(r));
    console.log('The volume of the cylinder is', hitungArea(r) * l);
    rl.close();
});

File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L5/04/Kasus2.py
import math

def hitung_area(r):
    return r * r * math.pi

def main():
    r, l = map(float, input("Enter the radius and length of a cylinder: ").split())
    
    print("The area is", hitung_area(r))
    print("The volume of the cylinder is", hitung_area(r) * l)

if __name__ == "__main__":
    main()

